### PR TITLE
add rasgo_drop_columns UDT

### PR DESCRIPTION
**Pull request by cpdough**

### CODE REVIEW

File: column_transforms/rasgo_drop_columns/rasgo_drop_columns.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  exclude_cols:
+    type: column_list
+    description: A list of the columns from the dataset you want to drop. Any columns not in the exclude_cols list will be kept.
+example_code: |
+
+    t1 = source.transform(
+        transform_name='rasgo_drop_columns',
+        include_cols=["DS_WEATHER_ICON", "DS_DAILY_HIGH_TEMP", "DS_DAILY_LOW_TEMP"]
+    )
+
```
Comment (szaghikian-rasgo) [inline column_transforms/rasgo_drop_columns/rasgo_drop_columns.yaml:22]: For examples where it's ultra simple, do we want to showcase usage of all options? Like you could have here:
```
    t2 = source.transform(
        transform_name='rasgo_drop_columns',
        exclude_cols=["Some_column", "some other column"]
    )
```

File: column_transforms/rasgo_drop_columns/rasgo_drop_columns.sql [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    {%- else -%}
+        {%- set database, schema, table = rasgo_source_ref(source_id).split('.') -%}
+    {%- endif -%}
+        SELECT COLUMN_NAME FROM {{ database }}.information_schema.columns
+        WHERE TABLE_CATALOG = '{{ database }}'
+        AND   TABLE_SCHEMA = '{{ schema }}'
+        AND   TABLE_NAME = '{{ table }}'
+{%- endmacro -%}
+
+{% if include_cols and exclude_cols is defined %}
```
Comment (szaghikian-rasgo) [inline column_transforms/rasgo_drop_columns/rasgo_drop_columns.sql:18]: does this need to be `if include_cols is defined and exclude_cols is defined`?

**Review by szaghikian-rasgo — APPROVED** [review]
Check the comment about `is defined` usage in the jinja template. Otherwise, LGTM

File: column_transforms/rasgo_drop_columns/rasgo_drop_columns.sql (hunk 2 above)
Comment (szaghikian-rasgo) [reply column_transforms/rasgo_drop_columns/rasgo_drop_columns.sql:18]: never mind. Looks like `is defined` is just a truthiness check so `if include_cols` == `if include_cols is defined`.

File: column_transforms/rasgo_drop_columns/rasgo_drop_columns.yaml (hunk 1 above)
Comment (cpdough) [reply column_transforms/rasgo_drop_columns/rasgo_drop_columns.yaml:22]: good idea - will add

**Pull request merged**